plugins/pipelines/remove.js

```diff
@@ -43,7 +43,7 @@ module.exports = () => ({
                     })
 
                     // user has good permissions, remove the pipeline
-                    .then(() => reply(pipeline.remove()));
+                    .then(() => reply(pipeline.remove()).code(204));
```

Comment:
Shouldn't this be something like:
`.then(pipeline.remove).then(() => reply().code(204));`